src/main/java/com/brew/oauth20/server/data/RefreshToken.java

```diff
@@ -37,10 +37,12 @@ public class RefreshToken {
     private OffsetDateTime expiresAt;
     @Column(name = "token", nullable = false, length = Integer.MAX_VALUE)
     private String token;
-    @Column(name = "revoked_at", nullable = true)
+    @Column(name = "scope", length = Integer.MAX_VALUE)
```

Comment:
we should remove length = Integer.MAX_VALUE here since it can cause problems later